test/spec.git-parser.js

```diff
@@ -9,42 +8,383 @@ describe('git-parser stash show', () => {
     const res = gitParser.parseGitStashShow(text);
     expect(res).to.be.an('array');
     expect(res.length).to.be(1);
-    expect(res[0].filename).to.be('New Text Document (2).txt');
+    expect(res[0]).to.eql({ filename: 'New Text Document (2).txt' });
   });
 });
 
-describe('git-parse diff on big change', () => {
-  describe('git-parser parseGitLog', () => {
-    it('should work with branch name with ()', () => {
-      const refs = gitParser.parseGitLog('commit AAA BBB (HEAD, (test), fw(4rw), 5), ((, ()')[0].refs;
+describe('git-parser parseDiffResult', () => {
+  it('all diff selected', () => {
+    var gitDiff = 'diff --git a/package.json b/package.json\n'
```

Comment:
can we stick with es6 `const` and `let`?

Reply:
done